CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 ## Latest
 
+## 0.6.4
+
+### Important
+- Following components will be deprecated and archived in next release:
```

Comment:
Have we confirmed the removal in the next release?  If no, I'd move the deprecation warning at the top to ensure it's visible to everyone reading the file

Reply:
Moved to deprecations section.